### DISCUSSION TITLE
BetterPlayerCacheConfiguration

### DISCUSSION THREAD

**Discussion by Khun-Andy** [jhomlala/betterplayer#556] [Q&A]
What happens when you init different video Data Sources with different maxCacheSize values? Or in other words, how to init or set the maxCacheSize and maxCacheFileSize parameters on the root level of the appl? Thank you so much.

**Comment by Khun-Andy** [comment]
The maximum cache size to keep on disk in bytes. This value is used only when first video access. cache. This value is used for all players within your app. It can't be changed during app work.

**Reply by jhomlala** [reply]
First setup of data source with cache settings will be used to determine cache size in `BetterPlayerController` instance.
